test/test-applications/integrations/Directory.Build.props

```diff
@@ -5,7 +5,7 @@
 
   <PropertyGroup>
       <!-- only run .NET Framework tests on Windows -->
-      <TargetFrameworks>net7.0;net6.0</TargetFrameworks>
+      <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
```

Comment:
please revert the order (it helps in VS code as the first one is used)

the same comment applies to other places

Reply:
https://github.com/signalfx/splunk-otel-dotnet/pull/94/commits/764a9dc6bbd81b09fa59db21d5ec0bf8c0a0ec27